left_finger_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: left_finger_joint
    joint_speed: 1.17
    joint_torque_limit: 0.6
    motor:
        id: 1
        init: 2048
        min: 0
        max: 4095

right_finger_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: right_finger_joint
    joint_speed: 1.17
    joint_torque_limit: 0.6
    motor:
        id: 2
        init: 2048
        min: 0
        max: 4095

elbow2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: elbow2_joint
    joint_speed: 1.17
    joint_torque_limit: 0.6
    motor:
        id: 3
        init: 2048
        min: 0
        max: 4095
